src/instructlab/model/evaluate.py

```diff
@@ -308,6 +319,11 @@ def launch_server(
     type=click.Path(),
     help="Path where all the MMLU Branch tasks are stored. Needed for running mmlu_branch.",
 )
+@click.option(
+    "--backend",
+    type=click.Choice([backends.VLLM, backends.LLAMA_CPP]),
```

Comment:
You can pass the frozenset directly.
```suggestion
    type=click.Choice(tuple(backends.SUPPORTED_BACKENDS)),
```